bodymass.py

```diff
@@ -0,0 +1,19 @@
+kilo = float (input("kilonuzu giriniz(örnek: 84.9): "))
+boy = float (input("Boyunuzu m cinsinden giriniz: "))
+
+vki = (kilo / (boy**2))
+
+if vki < 18.5:
+    print(f"vucut kitle indeksiniz:  {vki} zayıfsınız.")
+elif vki < 25:
+    print (f"vucut kitle indeksiniz: {vki} normalsiniz.")
```

Comment:
If using so many if-elif-else statements, then use match-case, which is much more better.